project.clj

```diff
@@ -3,8 +3,8 @@
   :license      {:name "MIT"
                  :url "https://github.com/CoNarrative/precept/blob/master/LICENSE"}
   :dependencies [[org.clojure/clojure "1.9.0-alpha15"]
```

Comment:
We probably want to bump this to alpha17 to keep in sync with Precept. https://github.com/CoNarrative/precept/blob/45a1912c05d71d79709cc347e9ab53b8fa48f6cb/project.clj#L6